CHANGELOG.md

```diff
@@ -14,6 +14,10 @@
     clients and then add the new scopes `ADMIN` and `ME`. See the
     [migration notes] (docs/Migration.md#from-22-to-30) for further details.
 
+- Allow colons (:) as field separators for URNs of extensions, since this is
+  what the SCIM specification defines. Using periods (.) is still possible,
+  but will log a warning message. 
+
```

Comment:
Most of the changelog entries take the form:

```
- <Some brief summary>

    <An optional, more descriptive text>
```

Don't get me wrong, I also like your style and I don't want to force usage of the old style or any style whatsoever. I'm just wondering whether we should agree on one style. As this is a fresh new changelog, anything is possible.